kadi/scripts/validate_states.py

```diff
@@ -68,15 +205,22 @@ def main(args=None):
     maude.conf.cache_msid_queries = True
     fetch.CACHE = True
 
-    if opt.in_work:
-        conf.include_in_work_command_events = True
+    kadi.commands.conf.include_in_work_command_events = opt.in_work
```

Comment:
In the unlikely event that a user has set the configuration in a file, the code now overrides that. I.e. the command line option wins.